analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/RoslynSymbolicExecutionTest.Branching.LearnConstraint.cs

```diff
@@ -602,6 +603,7 @@ public void Branching_LearnsObjectConstraint_RecursivePattern_ElseIsNull(string
         [DataRow("arg is TClass { }")]
         [DataRow("arg is TStruct { }")]
         [DataRow("arg is DateTime { Ticks: 0 }")]
+        [DataRow("arg is string { }", "T")]
```

Comment:
We had `[DataRow("arg is { }", "TStruct")]` so I also added `[DataRow("arg is { }", "int")]` there.

The `"arg is string { }", "TStruct"` doesn't compile, because
> CS8121 An expression of type 'TStruct' cannot be handled by a pattern of type 'string'.